sdk/websites/Azure.ResourceManager.AppService/src/Generated/WebSiteSlotResource.cs

```diff
@@ -6117,7 +6117,7 @@ public virtual Pageable<SlotDifference> GetSlotDifferencesSlot(CsmSlotEntity slo
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>WebApps_SwapSlot</description>
+        /// <description>WebApps_SwapSlotSlot</description>
```

Comment:
it's `SlotSlot`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/WebApps.json#L18318